### DISCUSSION TITLE
How to show first frame?

### DISCUSSION THREAD

**Discussion by Symbai** [SuRGeoNix/Flyleaf#309] [Q&A] [answered]
I'm opening a video with `AutoPlay = false` this always leaves me with a black screen. How can I make it so it shows the first frame?

**Comment by SuRGeoNix** [comment, marked as answer]
Hi @Symbai,

You could Seek to 0 for that. eg:-

```csharp
Player.OpenCompleted += (o, e) =>
{
	if (e.Success)
	{
		Player.Seek(0);
	}
};
```

**Reply by Symbai** [reply]
Thanks, works great
